package.json

```diff
@@ -28,6 +29,7 @@
     "lint-staged": "9.2.5",
     "node-fetch": "2.6.7",
     "npm-package-arg": "6.1.0",
+    "npm-run-all": "4.1.5",
```

Comment:
This package hasn't been updated in 5 years.

https://www.npmjs.com/package/npm-run-all

https://github.com/mysticatea/npm-run-all/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+vulnerability

Are you sure we need this?

Reply:
There's some other similar tools available. The repo has a thread about alternatives here: https://github.com/mysticatea/npm-run-all/issues/10

I wanted one that was npm-script-centric (i.e. don't need to specify "pnpm run …" in the child commands), so this one seemed to fit that.

Agreed about the unmaintained aspect. I didn't notice that. Open to suggestions for alternatives.

Reply:
Another possibility - Rust-based script invoker `dum`. Though it doesn't currently support serial/parallel execution, there is a tracking issue for it: https://github.com/egoist/dum/issues/18

Maybe one of the Rust aficionados on the team would want to implement support for that 😄

Reply:
I took a different approach and removed `npm-run-all` here: f08e46af883b261e6fb63bcfbc9bc3cb2d3e1be1